eve/main.yml

```diff
@@ -39,3 +41,34 @@ stages:
             npm run flow &&
             npm run lint
           haltOnFailure: false
+  post-merge:
```

Comment:
For now, there's no need to split your post-merge stage into 2, I would directly put the `storybook_deploy` stage like the following:
```
 development/*:
    stage: storybook_deploy
```

or rename the `storybook_deploy` as `post-merge`